common/backoff/jitter.go

```diff
@@ -29,36 +29,44 @@ import (
 	"time"
 )
 
-// JitDuration return random duration from (1-coefficient)*duration to (1+coefficient)*duration, inclusive, exclusive
-func JitDuration(duration time.Duration, coefficient float64) time.Duration {
-	validateCoefficient(coefficient)
+const fullCoefficient float64 = 1
 
-	return time.Duration(JitInt64(duration.Nanoseconds(), coefficient))
+// FullJitter return random number from 0 to input, inclusive, exclusive
+func FullJitter[T int64 | float64 | time.Duration](input T) T {
```

Comment:
with the suggested version, I think you can write this as:
```suggestion
func FullJitter[T ~int64 | ~int | ~float64](input T) T {
```